src/components/tables/DataTableCell.vue

```diff
@@ -44,10 +42,6 @@ export default {
   },
   methods: {
     transformDate: function(date) {
-      if (date === null) {
-        return '—';
-      }
-
       return moment(date).format('D MMM YYYY');
```

Comment:
what if the date is null? what is it going to show?

Reply:
@squallstar It will show '—'.
We have changed cell behavior on the template level. Know when a value is  `null`, `undefined` we will show a '—'.